Failure on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35439&attempt_id=56738f09-2545-44ba-9ca1-76817323c030)

<details><summary>Partial log (click to expand)</summary><p>

```
         | `Word of Core.String.t ] Patience_diff_lib__Patience_diff.Hunk.t
         list
       but an expression was expected of type
         'a Patdiff_lib.Import.Patience_diff.Hunks.t =
           'a Patience_diff_lib__Patience_diff.Hunk.t list

Waiting for 1 job to finish.
make: *** [Makefile:5: default] Error 1
builder for ‘/nix/store/df30fid1ygkpfd0cb3ly4fvqa39614mn-ocaml4.04.2-patdiff-0.10.0.drv’ failed with exit code 2
error: build of ‘/nix/store/df30fid1ygkpfd0cb3ly4fvqa39614mn-ocaml4.04.2-patdiff-0.10.0.drv’ failed
```
</p></details>

--- a/pkgs/tools/misc/patdiff/default.nix
+++ b/pkgs/tools/misc/patdiff/default.nix
@@ -4,7 +4,7 @@ with ocamlPackages;
 
 janePackage {
   name = "patdiff";
-  hash = "15b6nkmd2z07j4nnmkb2g6qn3daw2xmmz3lgswkj03v29ffib014";
+  hash = "04kl9h7j3pzpyic8p34b8i9vpf6qn7ixp077d8i44cpbymdqdn96";
   buildInputs = [ core_extended expect_test_helpers patience_diff ocaml_pcre ];
   meta = {
     description = "File Diff using the Patience Diff algorithm";

